R/eonet.R

```diff
@@ -26,53 +41,57 @@ earth_event <- function(status = "all",
                         days = 20,
                         LimitType = "limit",
                         TrySimplify = TRUE)
-  {
+{
 
-  h <- "http://eonet.sci.gsfc.nasa.gov/api/v2.1/events?"
+  events_url <- "http://eonet.sci.gsfc.nasa.gov/api/v2.1/events?"
```

Comment:
I'm not sure what you mean by that. I just renamed the variable `h` to `events_url` for better readability.